src/FakeItEasy/Configuration/IThenConfiguration.cs

```diff
@@ -0,0 +1,17 @@
+﻿namespace FakeItEasy.Configuration
+{
+    using System.Diagnostics.CodeAnalysis;
+
+    /// <summary>
+    /// Configures the next behavior for this call.
+    /// </summary>
+    /// <typeparam name="TInterface">The type of configuration interface to return.</typeparam>
+    public interface IThenConfiguration<out TInterface>
+    {
+        /// <summary>
+        /// Gets an object that lets you specify how the call will behave after the previous configuration has been consumed.
+        /// </summary>
+        [SuppressMessage("Microsoft.Naming", "CA1716:IdentifiersShouldNotMatchKeywords", MessageId = nameof(Then), Justification = "Part of the fluent syntax.")]

```

Comment:
I considered looking for another name, but the only language I can think of that uses `Then` (with this casing) as a keyword is VB.NET, and it seems to handle it just fine.